Module.yaml

```diff
@@ -0,0 +1,125 @@
+version: 1.0.0
+name: OPCUASubscribe
+description: OPC UA Subscribe connects to OPC UA Server and listens for incoming data messages and forwards the payload to next module.
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_OPCUA_input.png
+type: process
+tags:
+  - OPC UA Subscribe
+  - process
+categories:
+  - id: category
+    name: Process
+image:
+  name: weevenetwork/opcua-subscribe
+  tag: latest
+environments:
+  - key:  OPC_UA_SERVER
+    description: URL of OPC UA Server
+    value: ''
+    default: ''
+    type: string
+    name: OPC UA Server
+  - key:  REQUIRES_AUTHENTICATION
+    description: OPC UA Server requires authentication
+    value: ''
+    default: no
+    type: enum
+    options:
+      - yes
+      - no
+  - key:  OPC_UA_USERNAME
+    description: Username for authentication
+    value: ''
+    default: ''
+    type: string
+    name: OPC UA Username
+    dependencies:
+      - key: REQUIRES_AUTHENTICATION
+        value: yes
+  - key:  OPC_UA_PASSWORD
+    description: Password for authentication
+    value: ''
+    default: ''
+    type: string
+    name: OPC UA Password
+    dependencies:
+      - key: REQUIRES_AUTHENTICATION
+        value: yes
+  - key:  DISCARD_OLDEST
+    description: Discard oldest values
+    value: ''
+    default: true
+    type: enum
+    options:
+      - true
+      - false
+    name: Discard oldest
+  - key:  MAX_RETRY
+    description: Max retry for connecting
+    value: ''
+    default: 1
+    type: integer
+    name: Max retry
+  - key:  REQUESTED_PUBLISHING_INTERVAL
+    description: requestedPublishingInterval parameter
+    value: ''
+    default: 1000
+    type: integer
+    name: Requested publishing interval
+  - key:  REQUESTED_LIFETIME_COUNT
+    description: requestedLifetimeCount parameter
+    value: ''
+    default: 100
+    type: integer
+    name: Requested life time count
+  - key:  REQUESTED_MAX_KEEPALIVE_COUNT
+    description: requestedMaxKeepAliveCount parameter
+    value: ''
+    default: 10
+    type: integer
+    name: Requested max keep alive count
+  - key:  MAX_NOTIFICATIONS_PER_PUBLISH
+    description: maxNotificationsPerPublish parameter
+    value: ''
+    default: 100
+    type: integer
+    name: max notifications per publish
```

Comment:
please start the words with capital letters as it looks better in UI

Reply:
updated, my bad :)